### DISCUSSION TITLE
Redirect after successful sign in or sign up for credentials type

### DISCUSSION THREAD

**Discussion by jaekunchoi** [nextauthjs/next-auth#885] [Help] [answered]
**Your question**
Is there any examples and/or way to redirect to private dashboard on successful sign in or sign up for credentials type?
I couldn't find any clear docs around this

**What are you trying to do**
Trying to redirect after successful credentials type sign in and sign up

**Feedback**
*Documentation refers to searching through [online documentation](https://next-auth.js.org), code comments and issue history. The example project refers to [next-auth-example](https://github.com/iaincollins/next-auth-example).*

* [ ] Found the documentation helpful
* [x] Found documentation but was incomplete
* [ ] Could not find relevant documentation
* [ ] Found the example project helpful
* [x] Did not find the example project helpful

**Comment by meliborn** [comment]
Any news? I believe it must be essential feature to set after successful sign in path

**Comment by StefanSelfTaught** [comment, marked as answer]
You could try this:

![image](https://user-images.githubusercontent.com/43994418/109727034-55e3e980-7bbc-11eb-8bbd-6b34eccd2311.png)
